src/store.ts

```diff
@@ -279,6 +288,12 @@ export default function Store(
             this.SDKConfig.useCookieStorage = false;
         }
 
+        if (config.hasOwnProperty('usePersistence')) {
+            this.SDKConfig.usePersistence = config.usePersistence;
+        } else {
+            this.SDKConfig.usePersistence = false;
```

Comment:
This should be true by default.  Is this `false` because you are in the middle of development?

Reply:
Good catch. I made this default to `false` before we made our decision to default it to `true`. I'll update.